src/SwooleStream.php

```diff
@@ -102,35 +96,25 @@ public function getSize()
         return $this->bodySize;
     }
 
-    /**
-     * @return int
-     */
-    public function tell()
+    public function tell(): int
     {
         return $this->index;
     }
 
-    /**
-     * @return bool
-     */
-    public function eof()
+    public function eof(): bool
     {
         return $this->index >= $this->getSize();
     }
 
     /**
      * @return bool Always returns true.
      */
-    public function isReadable()
+    public function isReadable(): bool
     {
         return true;
     }
 
-    /**
-     * @param int $length
-     * @return string
-     */
-    public function read($length)
+    public function read(int $length): string
```

Comment:
We should probably not add parameter types, unless our lowest CI is checking for compatibility here

Reply:
Lowest does check compatibility with 1.1

Thankfully class is final so no inheritance BC to worry about here.